crates/state/src/bridge_state.rs

```diff
@@ -465,4 +476,32 @@ impl WithdrawOutput {
     pub fn destination(&self) -> &Descriptor {
         &self.destination
     }
+
+    pub fn amt(&self) -> BitcoinAmount {
+        self.amt
+    }
+}
+
+#[derive(Clone, Debug, Eq, PartialEq, BorshDeserialize, BorshSerialize, Serialize, Deserialize)]
+pub struct ExecutionState {
+    /// The index of the operator that has fronted the funds for the withdrawal,
+    /// and who will be reimbursed by the bridge notaries.
+    assignee: OperatorIdx,
+
+    /// Actual amount sent in withdrawal
+    amt: BitcoinAmount,
+}
```

Comment:
Does it maybe make sense for this to include the fulfillment txid?

Also as in the other comment, this would be `FulfilledState` which implies that it's waiting for reclamation.